source/_data/plugins.yml

```diff
@@ -2496,3 +2496,10 @@
     - server
     - editor
     - preview
+- name: hexo-cloudinary
```

Comment:
The name is not unique

![image](https://user-images.githubusercontent.com/16578570/75703860-96164080-5cb8-11ea-8409-166224a6a735.png)

Reply:
In fact the plugin already exists.

Reply:
@noraj Both of these plugins are developed by me.

This time I wanted to add `hexo-cldnry` plugin. I had to think of a unique name since `hexo-cloudinary` is taken already.

Reply:
Maybe `hexo-cloudinary-sdr`, or `hexo-cloudinary-responsive`. I prefer `hexo-cldnry`, though.

Reply:
Yeah use `hexo-cldnry` or `hexo-cloudinary-sdk` or whatever you prefer.